fury/actor.py

```diff
@@ -247,6 +258,138 @@ def sphere(
     )
 
 
+def ellipsoid(
+    centers,
+    *,
+    axes=None,
+    lengths=(4, 3, 2),
+    colors=(1, 0, 0),
+    opacity=None,
+    phi=16,
+    theta=16,
+    material="phong",
+    enable_picking=True,
+    smooth=True,
+):
+    """
+    Create ellipsoid actor(s) with specified orientation and scaling.
+
+    Parameters
+    ----------
+    centers : ndarray (N, 3)
+        Centers of the ellipsoids
+    axes : ndarray, shape (N, 3, 3) or (3, 3), optional
```

Comment:
Should we call it R?

Reply:
`Orientations` would be more suitable here. `axes` (referring to basis_axes) is also fine. `rotation_matrices` is also goos but very verbose same as `orientation_matrices`  but it's the best option here (orientation_matrices) . will change to `orientation_matrices` .